packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -2452,7 +2452,7 @@ async function handler(
 
     // Send Webhook call if hooked to BOOKING_CREATED & BOOKING_RESCHEDULED
     await handleWebhookTrigger({ subscriberOptions, eventTrigger, webhookData });
-  } else if (eventType.requiresConfirmation) {
+  } else {
```

Comment:
This isn't needed, there is no case when eventType.requiresConfirmation would be false in this else condition